hack/update-generated-crd.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+set -euo pipefail
+
+OUTDIR="${OUTDIR:-}"
+if [ -z "$OUTDIR" ]; then
+  OUTDIR="$(mktemp -d)"
+fi
+
+SKIP_COPY="${SKIP_COPY:-}"
```

Comment:
Should we treat set but empty as false? I have been treating set but empty as true.

Reply:
Fixed